analysis/wwz/get_wwz_yields.py

```diff
@@ -519,7 +633,8 @@ def make_syst_plots(histo_dict,grouping_mc,grouping_data,save_dir_path,year):
 
     for var_name in histo_dict.keys():
         #print(f"\n{var_name}")
-        if var_name not in TMP_VAR_LST: continue
+        if var_name != "njets": continue
```

Comment:
Let's not change the default behavior here

Reply:
fixed